part1.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+with open('wages.csv','r') as f:
+	file = f.readlines()
+
+
+
+
+for line in file:
+	line = line.split(',')
+	gender = line[0]
+	exp = line[1]
+	str = gender + ' ' + exp + '\n'
+	with open('new.txt', 'a') as n:
+		n.write(str)
+
```

Comment:
Missing the unique and sorting part of the question. 

data=pandas.read_csv("wages.csv")
genderYears=data.iloc[:,0:2]
unique_genderYears=genderYears.drop_duplicates()
sorted=unique_genderYears.sort_values(by=['gender', 'yearsExperience'])
sorted.to_csv('sortedUniqueWagesPython.txt',header=False,index=False,sep=' ')